compiler/rustc_session/src/options.rs

```diff
@@ -1465,6 +1465,9 @@ options! {
     #[rustc_lint_opt_deny_field_access("use `Session::print_llvm_passes` instead of this field")]
     print_llvm_passes: bool = (false, parse_bool, [UNTRACKED],
         "print the LLVM optimization passes being run (default: no)"),
+    #[rustc_lint_opt_deny_field_access("use `Session::print_llvm_stats` instead of this field")]
+    print_llvm_stats: bool = (false, parse_bool, [UNTRACKED],
```

Comment:
Maybe call it `print_codegen_stats` since it's forwarded to all ssa codegen backends